index.d.ts

```diff
@@ -3,4 +3,5 @@ interface Array<T> {
   Aggregate<T>(accum: (memo: T, val: T) => T): T;
   Select<T, U>(xform: (x: T) => U): Array<U>;
   Where<T>(predicate: (X: T) => boolean): Array<T>;
+  Last<T>():T;
```

Comment:
I haven't used typescript before. I hope that this works.

Reply:
Looks good to me. Great work!